test/cached-build.test.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+const { spawnSync } = require('child_process')
+const { readFileSync, readdirSync } = require('fs')
+const { resolve, join } = require('path')
+const { test } = require('tap')
+const webpack = require('webpack')
+const { banner, PinoWebpackPlugin } = require('../src/index')
+
+function runBuild(distFolder, onDone) {
+  webpack(
+    {
+      context: resolve(__dirname, 'fixtures'),
+      mode: 'production',
+      target: 'node',
+      entry: {
+        first: './first.js',
+        'abc/cde/second': './second.js'
+      },
+      output: {
+        path: distFolder,
+        filename: '[name]-[contenthash].js'
+      },
+      cache: {
+        name: 'test',
+        type: 'filesystem',
+        cacheDirectory: join(distFolder, 'cache'),
+        buildDependencies: {
+          config: [__filename]
+        }
+      },
+      plugins: [new PinoWebpackPlugin({ transports: ['pino-pretty'] })],
+      optimization: {
+        minimize: false
+      }
+    },
+    onDone
+  )
+}
+
+test('it should correctly generated all required pino files', (t) => {
```

Comment:
```suggestion
test('it should correctly generate all required pino files', (t) => {
```

Reply:
How is this testing that the output is being cached? It looks like it's testing that the output is generated correctly the second time you run the build, but not that the cache is being used. Am I missing something?

Reply:
The caching is enabled in the webpack configuration and I'm testing that nothing breaks when it is enabled.

I've renamed the test to make the intent more clear.